src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -269,17 +273,7 @@ def test_nav_links_webextensions(self):
         self.test_nav_links(show_compat_reporter=False)
 
 
-class TestEditBasicListed(BaseTestEditBasic):
-    __test__ = True
-
-    def test_edit_page_not_editable(self):
-        # The /edit page is the entry point for the individual edit sections,
-        # and should never display the actual forms, so it should always pass
-        # editable=False to the templates it renders.
-        # See https://github.com/mozilla/addons-server/issues/6208
-        response = self.client.get(self.url)
-        assert response.context['editable'] is False
-
+class TagTestsMixin(object):
```

Comment:
Some of these changes are reorganizing the different functionality into Mixins, so we can test Tags with Listed Static Themes, for example, but not the other add-on properties.